src/gui/application.cpp

```diff
@@ -247,6 +247,26 @@ Application::Application(int &argc, char **argv)
     // needed during commandline options parsing
     setApplicationVersion(_theme->versionSwitchOutput());
 
+#if defined(OC_PLUGIN_DIR) && defined(Q_OS_LINUX)
+    const QString extraPluginDir = QDir(QApplication::applicationDirPath()).filePath(QStringLiteral(OC_PLUGIN_DIR));
+    qCInfo(lcApplication) << "Adding extra plugin search path:" << extraPluginDir;
+    this->addLibraryPath(extraPluginDir);
+#endif
+
+    // Check vfs plugins
+    if (Theme::instance()->showVirtualFilesOption() && bestAvailableVfsMode() == Vfs::Off) {
+        qCWarning(lcApplication) << "Theme wants to show vfs mode, but no vfs plugins are available";
+    }
+    if (isVfsPluginAvailable(Vfs::WindowsCfApi))
+        qCInfo(lcApplication) << "VFS windows plugin is available";
+    if (isVfsPluginAvailable(Vfs::WithSuffix))
+        qCInfo(lcApplication) << "VFS suffix plugin is available";
+
+    if (_quitInstance) {
```

Comment:
`_quitInstance` will be set in parseOptions.

With the new parser however I'm pretty sure we can get rid of that bool and directly trigger ::quit from the parser?

Reply:
I should pay attention when moving code. This block is completely unrelated...